### PR TITLE
Added browser check bar on top of the window [v.2.5.2]

### PR DESCRIPTION
**Pull request by aklochkova**
Fixes https://github.com/OHDSI/Atlas/issues/1091

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,6 +23,13 @@
 </head>
 
 <body class="app">
+	<div class="browser-alarm" style="display:none;" data-bind="visible: isBrowserWarningShown()">
```
Comment (johnSamilin) [inline index.html]: why additional inline style here?

File: index.html [hunk 2]
```diff
@@ -32,8 +39,9 @@
 		</div>
 		<div data-bind="text:applicationStatus" id="status"></div>
 	</div>
+	<!-- /ko -->
 
-	<div style="display:none;" data-bind="visible: initializationComplete(), css: classes('menu-container')">
+	<div style="display:none;" data-bind="visible: initializationComplete(), css: classes(isBrowserWarningShown() ? 'menu-container-with-warning' : 'menu-container-without-warning')">
```
Comment (johnSamilin) [inline index.html]: use BEM

File: index.html [hunk 3]
```diff
@@ -57,7 +65,7 @@
 		</div>
 	</div>
 
-	<div id="wrapperMainWindow" style="display:none;" data-bind="visible:initializationComplete()">
+	<div id="wrapperMainWindow" style="display:none;" data-bind="visible:initializationComplete(), attr: {class: isBrowserWarningShown() ? 'mainWindowWithWarning' : 'mainWindowWithoutWarning'}">
```
Comment (johnSamilin) [inline index.html]: use BEM + follow agreement upon css classes naming (kebab-case)

File: index.html (hunk 1 above)
Comment (aklochkova) [reply index.html]: at this time knockout-bindings don't work yet

**Pull request merged**